fluentd/Dockerfile

```
@@ -43,10 +48,19 @@ RUN set -o xtrace && \
         /var/cache/debconf/* \
         /var/lib/apt/lists/* \
         /var/log/* \
```

Match indentation of newer lines
```suggestion
       /var/cache/debconf/* \
       /var/lib/apt/lists/* \
       /var/log/* \
```